src/processor/warning_context.cpp

```diff
@@ -0,0 +1,98 @@
+#include "processor/warning_context.h"
+
+#include "processor/result/factorized_table_util.h"
+
+using namespace kuzu::common;
+
+namespace kuzu {
+namespace processor {
+
+std::vector<std::string> WarningSchema::getColumnNames() {
+    static constexpr std::initializer_list<const char*> names = {"Message", "File Path",
```

Comment:
Move to constants.h as static constexpr.

Reply:
updated